### ISSUE TITLE
File temp path configurability in worflow.py

### ISSUE THREAD

**Issue by matentzn** [IHCC-cohorts/data-harmonization#97]
@jamesaoverton  is aware of this, but I just wanted to document it:

To make `src/workflow.py` work on my machine, I had to change

```
file_pattern = re.compile("/tmp/ring-multipart-\d+.tmp")
```

to

```
file_pattern = re.compile("/var/folders/5m/7qq8_hnd2fl38_yj341j4pwr0000gn/T/ring-multipart-\d+.tmp")
```

**Comment by beckyjackson** [comment]
I also made a fix for this in #94 but it may be too general.

**Comment by jamesaoverton** [comment]
@beckyjackson's change is good: `file_pattern = re.compile("/.+/ring-multipart-\d+.tmp")`

This is a hack around an issue with DROID: https://github.com/ontodev/droid/issues/49
